src/tests/unit/tests/DetailsView/components/left-nav/__snapshots__/left-nav-link-builder.test.tsx.snap

```diff
@@ -86,6 +86,392 @@ exports[`LeftNavBuilder buildAssessmentTestLinks should build links for assessme
 />
 `;
 
+exports[`LeftNavBuilder buildReflowAssessmentTestLinks should build links for assessments: getting started nav link render 1`] = `<GettingStartedNavLink />`;
+
+exports[`LeftNavBuilder buildReflowAssessmentTestLinks should build links for assessments: getting started nav link render 2`] = `<GettingStartedNavLink />`;
+
+exports[`LeftNavBuilder buildReflowAssessmentTestLinks should build links for assessments: requirement nav link render 1`] = `
+<TestViewLeftNavLink
```

Comment:
These massive snapshots are produced because we do `getElement` which renders/surfaces the passed properties as well. Since we validate the link objects ourselves, I almost prefer `debug` in this situation to eliminate additional coupling.

Would appreciate thoughts here.

Reply:
I personally prefer `debug()` for this situation

Reply:
I agree, debug() makes sense here since we validate the link objects anyway

Reply:
Well. I looked into using debug but unfortunately, it always seems to render the actual component (i.e. instead of <TestViewLeftNavLink> you see the actual html for that component).

I think this might be a part of the re-factoring work in all honesty: extract out the render functions to be separate components/modules that are not a part of this builder.